types/tar/index.d.ts

```diff
@@ -198,14 +201,17 @@ export const knownExtended: {
 export const headerSize: number;
 export const blockSize: number;
 
+/**
+ * A writable stream. Write tar data to it and it will emit entry events for each entry parsed from the tarball. This is used by tar.Extract.
+ */
+export const Parse: {
+  new (opt?: { file: string }): ParseStream;
```

Comment:
This doesn't seem to match up with the constructor options from the docs: https://github.com/npm/node-tar#constructoroptions-2

Reply:
Thank you! I fixed the points.